Formula/m/mapnik.rb

```diff
@@ -3,10 +3,9 @@ class Mapnik < Formula
   homepage "https://mapnik.org/"
   # needs submodules
   url "https://github.com/mapnik/mapnik.git",
-      tag:      "v4.0.2",
-      revision: "5f327ff3c88d8acca7c5db15b598258eea363aa7"
+      tag:      "v4.0.3",
```

Comment:
Skipping for now as conflict with https://github.com/Homebrew/homebrew-core/pull/193114 (EDIT: added as review comment to block CI).